src/component/Tooltip.tsx

```diff
@@ -113,6 +124,13 @@ function TooltipInternal<TValue extends ValueType, TName extends NameType>(props
    */
   const finalIsActive = activeFromProps ?? activeFromContext;
   const [lastBoundingBox, updateBoundingBox] = useGetBoundingClientRect(undefined, [payload, finalIsActive]);
+
+  const tooltipPortal = portalFromProps ?? tooltipPortalFromContext;
+  const cursorPortal = useCursorPortal();
+  if (tooltipPortal == null || cursorPortal == null) {
```

Comment:
what if someone doesn't want a cursor?

Reply:
The portal will be there always, the cursor inside will return null